airflow/www/app.py

```diff
@@ -120,6 +120,10 @@ def create_app(config=None, testing=False):
         )
         cookie_samesite_config = "Lax"
     flask_app.config["SESSION_COOKIE_SAMESITE"] = cookie_samesite_config
+    
```

Comment:
Could you remove whitespaces in this line.

You could do this manually, or by run `ruff` (if it installed), either:
- `ruff check --fix`
- `pre-commit run ruff --all-files` (if pre-commit installed)

Reply:
It's my mistake... I ran pre-commit.